File: Tugas2/java/pc.java
public class pc{
    private proc procpc;
    private Ram Rampc;
    private disk diskpc;
    private int total;

    // construct
    public pc() {}

    public pc(proc procpc, Ram Rampc, disk diskpc){
        this.procpc = procpc;
        this.Rampc = Rampc;
        this.diskpc = diskpc;
        this.total = procpc.getpricep() + Rampc.getpricer() + diskpc.getpriced();
    }
	
	//set and get
	public void setprocpc(proc procpc){
		this.procpc = procpc;
	}
	public proc getprocpc(){
		return this.procpc;
	}
	
	public void setRampc(Ram Rampc){
		this.Rampc = Rampc;
	}
	public Ram getRampc(){
		return this.Rampc;
	}
	
	public void setdiskpc(disk diskpc){
		this.diskpc = diskpc;
	}
	public disk getdiskpc(){
		return this.diskpc;
	}

    public int gettotal(){
        return this.total = procpc.getpricep() + Rampc.getpricer() + diskpc.getpriced();
    }

    public void printpc(){
        System.out.println("Prosesor PC :");
		procpc.printproc();
		System.out.println();
        System.out.println("RAM PC :");
		Rampc.printRam();
		System.out.println();
		System.out.println("Disk PC :");
		diskpc.printdisk();
		System.out.println();
		System.out.println("Total Harga : " + gettotal()); 
    }

}

File: Tugas2/python/pc.py
from proc import proc
from ram import ram
from disk import disk

class pc():
    __procpc = proc()
    __rampc = ram()
    __diskpc = disk()
    __total = 0

    # construct
    def __init__(self, procpc = proc(), rampc = ram(), diskpc = disk()):
        self.__procpc = procpc
        self.__rampc = rampc
        self.__diskpc = diskpc
        self.__total = procpc.getprice() + rampc.getpricer() + diskpc.getpricep()

    # set dan get
    def setprocpc(self, procpc):
        self.__procpc = procpc

    def getprocpc(self):
        return self.__procpc
    
    def setrampc(self, rampc):
        self.__rampc = rampc
    
    def getrampc(self):
        return self.__rampc

    def setdiskpc(self, diskpc):
        self.__diskpc = diskpc
    
    def getdiskpc(self):
        return self.__diskpc

    def gettotal(self):
        self.__total = self.__procpc.getprice() + self.__rampc.getpricer() + self.__diskpc.getpricep()
        total = self.__total
        return total

    # output
    def printpc(self):
        print("Prosesor PC : ")
        self.__procpc.printproc()
        print()
        print("RAM PC : " )
        self.__rampc.printram()
        print()
        print("Disk PC : ")
        self.__diskpc.printdisk()
        print()
        print("Total Harga : " + str(self.__total))